### PR TITLE
Update Overview dashboard

### PR DESCRIPTION
**Pull request by philipnrmn**
overview

### CODE REVIEW

**Comment by jieyu** [comment]
Thanks @philipnrmn. A few comments:
1) Instead of showing just number, I'd prefer we show a graph so that we can see the history of that number
2) Since this is DC/OS dashboard, we should be consistent with DC/OS naming. As a result, we should call agent "Node", and framework "Service"
3) Why the y axis of allocated resources graph a percentage? What does that mean?
4) Regarding resource allocation, i think we can show something similar to our UI does (one Graph per resource type)
5) I would suggest kill the Mesos tasks graph. Instead, we should defer to the Mesos dashboards.

File: dashboards/Overview.json [hunk 1]
```diff
@@ -63,576 +25,367 @@
...
+          "format": "time_series",
+          "intervalFactor": 1,
+          "legendFormat": "Containers",
+          "refId": "C"
+        }
+      ],
+      "thresholds": [],
+      "timeFrom": null,
+      "timeShift": null,
+      "title": "Running Containers",
```
Comment (jieyu) [inline dashboards/Overview.json]: We should change this given this Graph has more information than just contianers?

**Comment by philipnrmn** [comment]
screenshot from 2019-01-25 16-39-33

File: dashboards/Overview.json [hunk 2]
```diff
@@ -64,580 +52,617 @@
...
-      "gauge": {
-        "maxValue": 100,
-        "minValue": 0,
-        "show": false,
-        "thresholdLabels": false,
-        "thresholdMarkers": true
+      "thresholds": [],
+      "timeFrom": null,
+      "timeShift": null,
+      "title": "Running Tasks",
```
Comment (jieyu) [inline dashboards/Overview.json]: Maybe a better name for this given it also shows other numbers.

**Review by jieyu — APPROVED** [review]
One nit. The rest LGTM

**Comment by AmrAbdelrazik** [comment]
@philipnrmn  Is the CPU, Memory, Disk defined here allocation or utilization?

**Comment by philipnrmn** [comment]
Hi Amr, please try it out on the soak cluster! The graphs show both.

**Pull request merged**